### DISCUSSION TITLE
pipenv install textual is ok, but examples fail to display

### DISCUSSION THREAD

**Discussion by DavidTangye** [Textualize/textual#735] [General]
Eg the second example at https://github.com/Textualize/textual, namely -

```from textual.app import App
class ColorChanger(App):
    def on_key(self, event):
        if event.key.isdigit():
            self.background = f"on color({event.key})"
ColorChanger.run(log="textual.log")
```
results in a blank screen, (ctl-C to exit), and textual log contents of -
```---
driver=<class 'textual.drivers.linux_driver.LinuxDriver'>
Mount() >>> DockView(name='DockView#1')

```

(I am running manjaro linux on an  ODroid  ne2plus, 'linux-aarch64' platform)

**Comment by davep** [comment]
@DavidTangye I just tested this, installing with pipenv, creating a `test.py` and running it with `pipenv run python test.py` and it works okay here. Initially the screen is blank (black in my case as I use a black background terminal) -- if I then press any numeric key (note the `event.key.isdigit` in the example) the background colour changes to a colour related to that numeric value.

Are you doing the same? Running it up and then pressing numeric keys?

**Reply by DavidTangye** [reply]
Ah ... no! Thanks, I missed that. I could claim TLDR, but that probably shows I have the attention span of an ant :-)
Actually, I'm only 20 mins into the YT episode on Talk Python, https://www.youtube.com/watch?v=MN14DYgboOo adn got impatient to do a quick hack while trying to cook dinner.
